src/lib/commandline_cmds.ts

```diff
@@ -1,5 +1,23 @@
 import { messageOwnTab } from "@src/lib/messaging"
 import * as State from "@src/state"
+import { contentState } from "@src/content/state_content"
+
+const sleep = ms => new Promise((resolve) => setTimeout(resolve, ms))
+
+async function awaitProxyEq(proxy, a: string, b: string) {
+    return Promise.race(
+        [(async () => {
+            while (proxy[a] != proxy[b]) {
+                sleep(10)
```

Comment:
The `race` doesn't kill this once the race is over, which is not very surprising.

Simpler to just have a counter and give up once the counter exceeds some number.